### ISSUE TITLE
Wish `webpackChunkName` can support `(` and `)` like this: "user/(id)/page"

### ISSUE THREAD

**Issue by GiveMe-A-Name** [web-infra-dev/rspack#3157]
### What problem does this feature solve?

Rspack `WebpackChunkName` can't match string like this "user/(id)/page"

### What does the proposed API of configuration look like?

`WebpackChunkName` can match `(` and `)`

**Comment by suxin2017** [comment]
Please assign it to me

**Comment by suxin2017** [comment]
Is that what you're saying?
```js
import(/* webpackChunkName: "user/(id)/page"*/ './user/page/3')
```

output 
```
dist
--user
----(id)
------page.js
```
Or this one
output 
```
dist
--user
----user
------(id)
--------page
----------page.js
```

**Comment by hyf0** [comment]
Fixed by #3179.
